js/build-directory.js

```diff
@@ -237,6 +237,21 @@ DataDirectory.prototype.initialiseHandlebars = function() {
     });
   }
 
+  Handlebars.registerHelper('entry_thumbnail', function(entry) {
+    var thumbnail = entry[_this.config.thumbnail_field];
+
+    if (/api\.fliplet\.(com|local)/.test(thumbnail)) {
+      // attach auth token
+      thumbnail += (thumbnail.indexOf('?') === -1 ? '?' : '&') + 'auth_token=' + Fliplet.User.getAuthToken();
+    }
+    
+    if (!thumbnail) {
```

Comment:
You should probably run this check first, because if `!thumbnail` you don't need to add `auth_token`.

Reply:
You are right I was about to do it and then got distracted with a question from Ben.